x-pack/plugins/fleet/server/services/epm/registry/registry_url.ts

```diff
@@ -20,8 +20,8 @@ const SNAPSHOT_REGISTRY_URL_CDN = 'https://epr-snapshot.elastic.co';
 // const SNAPSHOT_REGISTRY_URL_NO_CDN = 'https://epr-snapshot.ea-web.elastic.dev';
 
 const getDefaultRegistryUrl = (): string => {
-  const branch = appContextService.getKibanaBranch();
-  if (branch === 'main') {
+  const isProduction = appContextService.getIsProductionMode();
+  if (!isProduction || branch === 'main') {
```

Comment:
you are missing the `const branch = appContextService.getKibanaBranch();` before

Reply:
Damnit. I knew I shouldn't just rely on editing files on Github 🙈
Thanks for the tip. Fixed.